### DISCUSSION TITLE
Additional Floor Effects Tracking

### DISCUSSION THREAD

**Discussion by pincoide** [Jukkales/NecroLens#10] [Ideas]
Here it is another idea for you: tracking the time limited floor effects like frailty, lethargy, witching, etc... so we can have a clear visible timer of how long the effect will last without wasting time checking the buffs :D

Basically the idea is to show 3 additional lines in the floor effects (with timer):
1. Witching
2. Rage/Petrify/Dread
3. Frailty/Lethargy (only for HoH/EO)

Additionally you could also add an optional more dramatic effect for the first 2 categories (like the cooldown effect) to make them even more noticeable when they are about to go off :)

**Comment by Jukkales** [comment]
more dramatic like a giant huge number counting down on screen? :D

It's an good idea yes. I will note it down.
Currently im thinking of a new little placeable window, which shows up the effect timers. 
Should be implemented fast.

If you have any ideas about feel free :)

**Reply by pincoide** [reply]
yeah the countdown effect I meant, so you know when to stop :D

A separate window could also work especially if it's something like the gather window from gatherbuddy :)
